### PR TITLE
Add tags for blog posts

### PR DESCRIPTION
**Pull request by Subhraprakash2020**
- Add tags for blog posts.
 - First create a tag table.
 - Join tag table with post table with many to many relation.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -50,7 +50,7 @@ gem "bootsnap", require: false
 
 group :development, :test do
   # See https://guides.rubyonrails.org/debugging_rails_applications.html#debugging-with-the-debug-gem
```
Comment (gajendrajena) [inline Gemfile:52]: need to remove this comment as we removed the debuger gem

File: app/controllers/posts_controller.rb [hunk 2]
```diff
@@ -17,10 +17,14 @@ def new
...
   # GET /posts/1/edit
   def edit
+    @tags = Tag.all.map { |tags| [tags.name, tags.id] }
+
   end
 
   # POST /posts or /posts.json
   def create
+    
+
```
Comment (gajendrajena) [inline app/controllers/posts_controller.rb:27]: remove empty lines

File: app/helpers/tags_helper.rb [hunk 3]
```diff
@@ -0,0 +1,2 @@
+module TagsHelper
```
Comment (gajendrajena) [inline app/helpers/tags_helper.rb:1]: remove this file.

File: app/views/posts/_form.html.erb [hunk 4]
```diff
@@ -22,7 +22,23 @@
...
     <%= form.submit %>
   </div>
+
+  <script>
+   $(".mul-select").select({
+      placeholder: "select Option",
+      tags: true,
+    });
+
+</script>
```
Comment (gajendrajena) [inline app/views/posts/_form.html.erb:43]: move the javascript to a javascript file

**Review by gajendrajena — CHANGES_REQUESTED** [review]

File: app/javascript/application.js [hunk 5]
```diff
@@ -1,3 +1,8 @@
 // Configure your import map in config/importmap.rb. Read more: https://github.com/rails/importmap-rails
 import "@hotwired/turbo-rails"
 import "controllers"
+$(".mul-select").select({
+   placeholder: "select Option",
+   tags: true,
+ });
+
```
Comment (gajendrajena) [inline app/javascript/application.js:8]: should be in dom ready, or else it may not work.

Comment (Subhraprakash2020) [reply app/javascript/application.js:8]: It work.

**Review by gajendrajena — APPROVED** [review]

**Pull request merged**